### ISSUE TITLE
[Website] Update schema explorer to show ToggleVisibility as deprecated

### ISSUE THREAD

**Issue by anna-dingler** [microsoft/AdaptiveCards#8765]
# Issue

It should be more clear on our website that ToggleVisibility is a deprecated control for accessibility reasons. We should update the schema explorer to show this change.

**Comment by joshdunnlime** [comment]
For which version?

**Comment by anna-dingler** [comment]
@joshdunnlime Please see this blog post:  https://adaptivecards.io/blog/2022/Community-Call-July/

This is a non-breaking change as ToggleVisibility is still available. However, it is no longer widely supported, specifically surrounding accessibility.
